display/d.text/test.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+import math
+import re
+
+# fmt: off
+fonts = ("cyrilc", "gothgbt", "gothgrt", "gothitt", "greekc", "greekcs",
+         "greekp", "greeks", "italicc", "italiccs", "italict", "romanc",
+         "romancs", "romand", "romans", "romant", "scriptc", "scripts",
+         "cyrilc", "gothgbtlu")
+colors = ("red", "orange", "yellow", "green", "blue", "indigo", "violet",
+          "black", "gray", "brown", "magenta", "aqua", "grey", "cyan",
+          "purple")
+# fmt: on
+
+
+def rc(*args):
+    print(f".X {args[0]}\n.Y {args[1]}")
+
+
+def xy(*args):
+    print(f".X {args[0]}\n.Y {args[1]}")
```

Comment:
I had to add `{args[0] / 100}` for something to show up. The original code has `%`. I'm not sure how it should work. It does not work for me. I did this to see it:

```
# in a GRASS session
d.mon wx0
python display/d.text/test.py > test.txt
d.erase
d.text input=test.txt
```

Whether it works for you or not, I think the ` / 100` is needed here.

Reply:
This one is interesting. It shows up for me the same without either dividing by 100 or without %, which is why I had missed the % being left out.

I was testing with the following, not that it should matter too much.
```
d.text >  d.mon start=wx0
./test.py | d.text
```

Reply:
I see the rotated colorful font names only when I do `/ 100` not when I use `%` in the code. The result I see is not supper nice either way (see below), but the current code does not give me the colorful font names. Are you getting nicer outputs @HuidaeCho @ocsmit?

![Screenshot from 2021-03-11 22-53-05](https://user-images.githubusercontent.com/5449060/110889669-99291100-82bc-11eb-80aa-148c2c82fd61.png)

Reply:
It's supposed to look like this:
![image](https://user-images.githubusercontent.com/7456117/110963256-da9ed800-831f-11eb-8e6c-b18996d34106.png)

```bash
d.mon start=png
./test.pl | d.text
d.mon stop=png
```

The reason why you cannot see the rotating font names with `%` is the wx0 monitor and GUI start `at=50,50` (#1441). In fact, the behavior of wx0 and GUI is inconsistent with the other monitors. Starting from the top-left corner used to be and should be the default.

Reply:
I managed to get something closer to the original perl script while compensating for the newer start point with 
```python
def xy(x, y):
    print(f".X {x - 50}%\n.Y {y + 25}%")
```
![image](https://user-images.githubusercontent.com/55674113/110993539-6bd47580-8345-11eb-8bec-e07f341219ec.png)

Obviously it is still not the same as the original but its closer to the intended output while appearing much more condensed.

Reply:
Using the following
```bash 
d.mon start=wx3                    
./test.py | d.text at=1,100
```
I get 
![image](https://user-images.githubusercontent.com/55674113/111001852-8cef9300-8352-11eb-9ce0-c9d35100601c.png)
without changing any of the code.

Reply:
> without changing any of the code.

The code was meant to test `%`, so it should include `%`.

Reply:
>without changing any of the code.

This was in reference to c714586c8dc596826e9bf8783d697a8345f406bf, which has percents added. I should have been more specific or created a different comment.

Reply:
Beautiful! Well, not really, but it does what it is supposed to do :-)